src/CoreWCF.NetTcp/src/CoreWCF/Security/MessageSecurityOverTcp.cs

```diff
@@ -0,0 +1,140 @@
+                        break;
+                    case MessageCredentialType.IssuedToken:
+                        throw new NotImplementedException();
+                       // oneShotSecurity = SecurityBindingElement.CreateIssuedTokenOverTransportBindingElement(IssuedSecurityTokenParameters.CreateInfoCardParameters(new SecurityStandardsManager(), this.algorithmSuite));
+                    default:
+                        Fx.Assert("unknown ClientCredentialType");
+                        throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new NotSupportedException());
+                }
+                result = SecurityBindingElement.CreateSecureConversationBindingElement(oneShotSecurity);
+            }
+            else
+            {
+                throw new NotSupportedException();
+               /* switch (this.clientCredentialType)
+                {
+                    case MessageCredentialType.None:
+                        oneShotSecurity = SecurityBindingElement.CreateSslNegotiationBindingElement(false, true);
+                        break;
+                    case MessageCredentialType.UserName:
+                        // require cancellation so that impersonation is possible
+                        oneShotSecurity = SecurityBindingElement.CreateUserNameForSslBindingElement(true);
+                        break;
+                    case MessageCredentialType.Certificate:
+                        oneShotSecurity = SecurityBindingElement.CreateSslNegotiationBindingElement(true, true);
+                        break;
+                    case MessageCredentialType.Windows:
+                        // require cancellation so that impersonation is possible
+                        oneShotSecurity = SecurityBindingElement.CreateSspiNegotiationBindingElement(true);
+                        break;
+                    case MessageCredentialType.IssuedToken:
+                        oneShotSecurity = SecurityBindingElement.CreateIssuedTokenForSslBindingElement(IssuedSecurityTokenParameters.CreateInfoCardParameters(new SecurityStandardsManager(), this.algorithmSuite), true);
+                        break;
+                    default:
+                        Fx.Assert("unknown ClientCredentialType");
+                        throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new NotSupportedException());
+                }
+                result = SecurityBindingElement.CreateSecureConversationBindingElement(oneShotSecurity, true);*/
+            }
+
+            // set the algorithm suite and issued token params if required
+            result.DefaultAlgorithmSuite = oneShotSecurity.DefaultAlgorithmSuite = this.AlgorithmSuite;
+
+            result.IncludeTimestamp = true;
+            if (!isReliableSession)
+            {
+                result.LocalServiceSettings.ReconnectTransportOnFailure = false;
+               // result.LocalClientSettings.ReconnectTransportOnFailure = false;
+            }
```

Comment:
Remove this and the other commented out code referencing LocalClientSettings as they are client only api's and won't ever be brought over.